app/helpers/rad_common/application_helper.rb

```diff
@@ -13,7 +13,10 @@ def secured_link(resource, format: nil)
     end
 
     def show_route_exists_for?(record)
-      Rails.application.routes.url_helpers.respond_to? "#{record.class.table_name.singularize}_path"
```

Comment:
this was not taking into consideration the :get method so that a route that had update, destroy, etc but no show route would still return true